integration-test-common/src/test/java/org/apache/gravitino/integration/test/container/RangerContainer.java

```diff
@@ -40,15 +40,18 @@ public class RangerContainer extends BaseContainer {
   public static final int RANGER_SERVER_PORT = 6080;
   public RangerClient rangerClient;
   private String rangerUrl;
-  private static final String username = "admin";
-  // Apache Ranger Password should be minimum 8 characters with min one alphabet and one numeric.
-  private static final String password = "rangerR0cks!";
 
   /**
-   * for kerberos authentication: authType = "kerberos" username = principal password = path of the
-   * keytab file
+   * for kerberos authentication: <br>
+   * authType = "kerberos" <br>
+   * username = principal <br>
+   * password = path of the keytab file <br>
    */
-  private static final String authType = "simple";
+  public static final String authType = "simple";
+
+  public static final String rangerUserName = "admin";
+  // Apache Ranger Password should be minimum 8 characters with min one alphabet and one numeric.
```

Comment:
```suggestion
  // Apache Ranger Password should have the character size bigger than 8  with at least one alphabet and one numeric.
```

Reply:
This is Apache Ranger origion comments, https://github.com/apache/ranger/blob/master/security-admin/scripts/install.properties#L84

Reply:
OK.